core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1739,8 +1743,8 @@ object SnappySession extends Logging {
             plan.childRDD.id, true)
       case _: LocalTableScanExec =>
         (null, Array.empty[Int], -1, false) // cache plan but no cached RDD
-      case _ if allbroadcastplans.nonEmpty =>
-        (null, Array.empty[Int], -1, false) // cache plan but no cached RDD
+      // case _ if allbroadcastplans.nonEmpty =>
+        // (null, Array.empty[Int], -1, false) // cache plan but no cached RDD
```

Comment:
So now you are using bchj mismatch instead of above to avoid caching? Does the bchj mismatch approach still result in caching albeit separate for each token value like was done without tokenization changes?